src/tests/cucumber/features/modeler-features/epic2/test_launcher_6.feature

```diff
@@ -0,0 +1,39 @@
+Feature: 6 - Simple end-to-end tests to test port management
```

Comment:
ctrl+alt+L in clion or pycharm would improve the formatting of the tables

Reply:
I did that, nothing changed. Sorry.